src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -322,16 +322,27 @@ public static function getClassDescription(string $class, string &$resolvedClass
 
     private function getContainerEnvVars(ContainerBuilder $container): array
     {
+        $bag = $container->getParameterBag();
+        $getDefaultParamter = function (string $name) {
+            return parent::get($name);
+        };
+        $getDefaultParamter = $getDefaultParamter->bindTo($bag, \get_class($bag));
+
+        $file = file_get_contents($container->getParameter('debug.container.dump'));
+
+        preg_match_all('{%env\((.*)\)%}', $file, $envVars);
```

Comment:
a default `key="env(x)"` should still be captured no? I believe the regex from @nicolas-grekas is OK.

Reply:
why should it be? matching it from parameters would mean we consider them as real env var references, isn't it?
I agree my regexp should be used otherwise :)

Reply:
indeed, we get the default using the closure bindto trick already.